R/addRank.R

```diff
@@ -0,0 +1,61 @@
+#' Rank the sample information in a `SummarizedExperiment` Object
+#'
+#' The information stored in `colData` in \linkS4class{SummarizedExperiment}
+#' object is ranked by `addRank` function and newly ranked field is added back
+#' to \linkS4class{SummarizedExperiment} object.
+#'
+#' @param x \linkS4class{SummarizedExperiment} object
+#' @param field A character value indicating the name of the matrix column that
+#' is aimed to be ranked.
+#' @param field_matrix A matrix that is aimed to be ranked. If the aimed matrix
+#' string already exists in colData segment, this optional matrix doesn't need
+#' to be added.
+#' @param rank_field_name A character value to name the newly ranked `field`
+#' @param ... Allow new parameters to be defined for this function.
+#'
+#' @return \linkS4class{RangedSummarizedExperiment} object with ranked field
+#' added
+#'
+#' @examples
+#' library(SummarizedExperiment)
+#' data(airway, package="airway")
+#' se <- airway
```

Comment:
We could prefer our own example data sets but we can leave this for later because mia pkg demo data sets do not have time series currently (we could add?), and using microbiomeDataSets is slow as it has to download data over network